app/controllers/api/v2/usergroups_controller.rb

```diff
@@ -22,6 +22,7 @@ def show
       def_param_group :usergroup do
         param :usergroup, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :admin, :bool, :required => false, :desc => N_("is an admin group-account")
```

Comment:
The "group-account" in the description is unclear to me, I'd suggest "is an admin user group", "has admin access" or similar.